dbclient/JobsClient.py

```diff
@@ -178,8 +179,19 @@ def adjust_ids_for_cluster(settings): #job_settings or task_settings
                 if 'format' not in job_settings or job_settings.get('format') == 'SINGLE_TASK':
                     adjust_ids_for_cluster(job_settings)
                 else:
+                    mod_task_settings = []
                     for task_settings in job_settings.get('job_clusters', []):
                         adjust_ids_for_cluster(task_settings)
+                    if mod_task_settings:
```

Comment:
Will this ever evaluate to true, since it's being set to [] in line 182?

Reply:
see [here](https://stackoverflow.com/a/53522/3519000)
an empty list returns a False in python
but yes we could use `len(mod_task_settings) > 0` to make it more clear

Reply:
added a comment that hopefully is more clear